app/src/main/java/com/example/warehousemanagement/domain/model/Status.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.warehousemanagement.domain.model
+
+enum class Status(val value: String) {
```

Comment:
Why `NotificationType` do not have parameter value like this. ( Chọn 1 trong 2 cho đồng bộ)

Reply:
cái chỗ này nè